app/src/services/chefs.ts

```diff
@@ -93,6 +93,7 @@ const service = {
             submissionId: response.submission.id,
             applicationStatus: APPLICATION_STATUS_LIST.NEW,
             confirmationId: response.submission.confirmationId,
+            companyNameRegistered: response.submission.companyNameRegistered,
```

Comment:
i think companyNameRegistered is available at `submission.companyNameRegistered` .
I see some of the data returned from chefs is in `submission` (eg createdAt), and actual form fields nested in `submission.data`. it might be worth flattening all the attributes into one layer further up the stack to avoid confusion.

Reply:
Good catch, thanks.